en/components/carousel.md

```diff
@@ -341,7 +341,41 @@ These configurions will have the following result:
 </div>
 
 
-
+## Accessibility
+### WAI-ARIA Roles, States, and Properties
+ * The Carousel base element role is [`region`](https://www.w3.org/TR/wai-aria-1.1/#region) - section containing content that is relevant to specific purpose and users will likely want to be able to navigate easily.
+ * Carousel indicators are with role [`tab`](https://www.w3.org/TR/wai-aria-1.1/#tab) -  grouping label providing a mechanism for selecting the tab content that is to be rendered to the user
+ * The element that serves as the container for the set of tabs (carousel indicators) role is set to [`tablist`](https://www.w3.org/TR/wai-aria-1.1/#tab).
+ * Each slide element is set with role [`tabpanel`](https://www.w3.org/TR/wai-aria-1.1/#tabpanel).
+ * The element that serves as the container for the set of igx-slides is set with [aria-live](https://www.w3.org/TR/wai-aria-1.1/#aria-live)="polite". Both options are
+   - `off`: if the carousel is automatically rotating.
+   - `polite`: if the carousel is NOT automatically rotating.
+
+### ARIA support
+#### **Carousel component**
+##### `Roles`: 
+ * [role="region"](https://www.w3.org/TR/wai-aria-1.1/#region)
+ * [role="button"](https://www.w3.org/TR/wai-aria/roles#button)
+ * [role="tablist"](https://www.w3.org/TR/wai-aria-1.1/#tablist)
+ * [role="tab"](https://www.w3.org/TR/wai-aria-1.1/#tab)
+##### `Attributes`:   
```

Comment:
need a new line here otherwise, the text is displayed li this
![Screenshot_2](https://user-images.githubusercontent.com/34240583/106742685-05618680-6626-11eb-89c4-d6677158f482.png)